src/main/java/TaskList.java

```diff
@@ -0,0 +1,49 @@
+import java.util.ArrayList;
+
+/**
+ * Duke's TaskList. Contains the task list and has operations to modify the list.
+ */
+public class TaskList {
+    ArrayList<Task> taskList;
```

Comment:
Might be better to use `tasks` as variable name, might confuse yourself with the `TaskList` class.

Reply:
agree:)